src/odemis/gui/comp/fastem_project_manager_panel.py

```diff
@@ -123,6 +125,25 @@ def is_unique(hls_color):
             return new_color_hex
 
 
+@dataclass
+class ImportData:
```

Comment:
This name is really generic. I'd suggest to name it maybe like `ImportedDataRow`... but feel free to find a better name!